android/Showcase/src/main/java/com/spruceid/mobilesdkexample/wallet/HandleMdocOID4VPView.kt

```diff
@@ -243,9 +245,10 @@ fun MdocFieldSelector(
             textAlign = TextAlign.Center
         )
 
+        val columnMod = if (fillHeight) Modifier.fillMaxSize() else Modifier
```

Comment:
This is not usual. It's better to accept the `modifier: Modifier = Modifier` as parameter and use this if necessary. In this case, you will only need to change the MdocFieldSelector in the `HandleMdocOID4VPView.kt`to
```
MdocOID4VPState.SelectiveDisclosure ->
            MdocFieldSelector(
                match = selectedMatch!!,
                onContinue = {  ... },
                modifier = Modifier.fillMaxSize(),
                onCancel = { onBack() }
            )
```